data/gegno/gegno outfits.txt

```diff
@@ -0,0 +1,785 @@
+	plural "R02 Battlezone Batteries"
+	cost 32000
+	mass 23
+	thumbnail outfit/gbatterysmall
+	"energy capacity" 2237
+	"outfit space" -23
+	description `Battlezone batteries are primarily used on the Gegno's medium-range combat vessels, holding a moderate amount of power for long lasting engagements. They also fit nicely on larger vessels as a buffer to power their larger systems.`
+
+
+outfit "R03 Warforge Battery"
+	category Power
+	licenses
+		"Gegno Civilian"
+	plural "R03 Warforge Batteries"
+	cost 61700
+	mass 41
+	thumbnail outfit/gbatterymed
+	"energy capacity" 4469
+	"outfit space" -41
+	description `Warforge batteries have proved their usefulness over the years as large, sturdy battery packs. They support the equally large volleys that are fired from Gegno ships, and help maintain an excess of power for engine usage.`
+
+
+outfit "R04 Crusade Battery"
+	category Power
+	licenses
+		"Gegno Civilian"
+		"Vi Evocati"
+		"Vi Centurion"
+		"Vi Lord"
+	plural "R04 Crusade Batteries"
+	cost 136000
+	mass 83
+	thumbnail outfit/gbatterylarge
+	"energy capacity" 8885
+	"outfit space" -83
+	description `As the largest battery mounted on Gegno ships, the Crusade Battery holds a large amount of energy, more that it would take a small generator a decent amount of time to produce in order to support the energy hungry warships of both the Scin and the Vi. However, the Gegno are inefficient at increasing density with size, causing these batteries to fall behind similarly sized options.`
+
+
+outfit "C3 Scrap Cell"
+	category Power
+	licenses
+		"Gegno Civilian"
+	cost 54000
+	mass 25
+	thumbnail outfit/vicoretiny
+	"energy generation" 1.4
+	"heat generation" 2.2
+	"outfit space" -25
+	description `The Scrap Cell is a small radiothermal generator that produces a limited amount of energy. It is only used in tandem with the Gegno's smaller-sized vessels and internals.`
+
+
+outfit "C7 Brawl Cell"
+	category Power
+	licenses
+		"Gegno Civilian"
+	cost 68100
+	mass 39
+	thumbnail outfit/vicoresmall
+	"energy generation" 2.5
+	"heat generation" 3.9
+	"outfit space" -39
+	description `Unlike other C-class cores, C7 Brawl Cells are mounted horizontally. This design, however, still serves the Gegno well, though it will likely be phased out in favor of larger generators becoming more compact and efficient in the future.`
+
+
+outfit "C17 Warzone Core"
+	category Power
+	licenses
+		"Gegno Civilian"
+	cost 2472000
+	mass 87
+	thumbnail outfit/vicoremed
+	"energy generation" 14.1
+	"heat generation" 38.3
+	"outfit space" -87
+	description `As did humans, the Gegno began developing nuclear technology once their cities grew massive enough to require it. The Warzone Core is a medium-sized nuclear fission reactor that is a suitable power source for most of the Gegno's large ships, though it looks even more discomforting than a human one.`
+
+
+outfit "C27 Campaign Core"
+	category Power
+	licenses
+		"Gegno Civilian"
+		"Vi Evocati"
+		"Vi Centurion"
+		"Vi Lord"
```

Comment:
these licenses feel a bit weird to me, why require all 4 or just the civilian? am I missing smth?
Are these meant to be different ways to prove yourself? I just don't see the added benefit of having these 3 be separate instead of being a license that would bring the same as all 3

Reply:
Yes it's a lore thing. Civilian means access to the basic tech and ships. Vi Evocati means access to basic Vi tech. Centurion, intermediary tech, Lord, strongest tech. They are a kind of "ranking" system. 

They come from these Wiki definitions, and are used within Vi society:

An evocatus (plural evocati) was a soldier in the Ancient [Roman army](https://en.wikipedia.org/wiki/Roman_army) who had served out his time and obtained an honorable discharge ([honesta missio](https://en.wikipedia.org/wiki/Honesta_missio)) but had voluntarily enlisted again at the invitation of the [consul](https://en.wikipedia.org/wiki/Roman_consul) or other commander.

A centurion ([/sɛnˈtjʊəriən/](https://en.wikipedia.org/wiki/Help:IPA/English); [Latin](https://en.wikipedia.org/wiki/Latin_language): centurio [[kɛn̪ˈt̪ʊrioː]](https://en.wikipedia.org/wiki/Help:IPA/Latin), pl. centuriones; [Greek](https://en.wikipedia.org/wiki/Greek_language): κεντυρίων, [translit.](https://en.wikipedia.org/wiki/Romanization_of_Greek) kentyríōn, or Greek: ἑκατόνταρχος, [translit.](https://en.wikipedia.org/wiki/Romanization_of_Greek) hekatóntarkhos) was a position in the [Roman army](https://en.wikipedia.org/wiki/Roman_army) during [classical antiquity](https://en.wikipedia.org/wiki/Classical_antiquity), nominally the commander of a [century](https://en.wikipedia.org/wiki/Centuria) ([Latin](https://en.wikipedia.org/wiki/Latin_language): centuria), a [military unit](https://en.wikipedia.org/wiki/Military_unit) of around 80 [legionaries](https://en.wikipedia.org/wiki/Legionaries).

Lord is an appellation for a person or [deity](https://en.wikipedia.org/wiki/Deity) who has [authority](https://en.wikipedia.org/wiki/Authority), control, or [power](https://en.wikipedia.org/wiki/Power_(social_and_political)) over others, acting as a master, chief, or ruler.[[](https://en.wikipedia.org/wiki/Lord#cite_note-1)

Reply:
So I guess you'll change the outfit licenses? Caus currently they all require all 3 its a bit weird

Reply:
Like I said above, that's intentional. First, you have to prove yourself as a Gegno Civilian someday, before you can do anything with factions that are warring in their space. Once you've done that, you're required to go up the ranks on each respective faction for the given tech. Some requires some of the licenses, some requires all of them, etc.. It's all relative to the tech you're trying to get.

Reply:
If something includes both weapon tech and advanced technology tech, for example, both Scin licenses are required for that Scin tech. Scin ranks are more mix-match for unlocks, and Vi ones are more progressive.

Reply:
Seems good
Mb all military outfits are together and so thats why I thought that all outfits had all 3 licenses
I feel like gegno could be a good place to start the game in honestly